packages/zipkin/README.md

```diff
@@ -3,6 +3,8 @@
 This is the core npm package for Zipkin. It contains the public API which is used by the various
 plugins (instrumentations and transports).
 
+We include TypeScript [definition file](index.d.ts) which you can also use a documentation.
```

Comment:
'use a documentation' or did you mean use as documentation?

Reply:
I did mean "use as documentation"